guides/getting_started.md

```diff
@@ -111,18 +98,7 @@ You can make interfaces automatically refresh by setting `live=True` in the inte
 
 Note there is no submit button, because the interface resubmits automatically on change.
 
-### Using State
-
-Your function may use data that persists beyond a single function call. If the data is something accessible to all function calls and all users, you can create a global variable outside the function call and access it inside the function. For example, you may load a large model outside the function and use it inside the function so that every function call does not need to reload the model.
-
-Another type of data persistence Gradio supports is session **state**, where data persists across multiple submits within a page load. However, data is *not* shared between different users of your model. To store data in a session state, you need to do three things: (1) Pass in an extra parameter into your function, which represents the state of the interface. (2) At the end of the function, return the updated value of the state as an extra return value (3) Add the `'state'` input and `'state'` output components when creating your `Interface`. See the chatbot example below: 
-
-{{ code["chatbot"] }}
-{{ demos["chatbot"] }}
-
-Notice how the state persists across submits within each page, but the state is not shared between the two pages. Some more points to note: you can pass in a default value to the state parameter, which is used as the initial value of the state. The state must be a something that can be serialized to a JSON format (e.g. a dictionary, a list, or a single value. Typically, objects will not work).  
-
-### Flagging
+### Flagging 🚩
```

Comment:
Hmm I wouldn't want users to miss these...

To make the quickstart easier to navigate, I'm adding a sidebar similar to the docs

Reply:
If that is the case I suggest we move these sections to the end of the document. Most of the users will take a quick look, and skim through.

Reply:
agree with @FarukOzderim. Live doesn't need to be a section, it can be a single sentence.

Reply:
@abidlabs pointing this comment in case you've not seen it:

> If that is the case I suggest we move these sections to the end of the document. Most of the users will take a quick look, and skim through.